doc-content/enterprise-only/integration/entando-integrating-proc.adoc

```diff
@@ -1,34 +1,11 @@
 [id='entando-integrating-proc_{context}']
 
-= Configuring {KIE_SERVERS} in Entando App Builder
+= Configuring {PRODUCT} with Entando App Builder
 
-In order to integrate your {PRODUCT} data in Entando, you must first configure one or more {KIE_SERVERS} in Entando App Builder. The configured {KIE_SERVERS} must be the servers to which you deployed the projects that you created in {PRODUCT}.
+In order to integrate your {PRODUCT} data in App Builder, you must first configure one or more {KIE_SERVERS} in App Builder. The configured {KIE_SERVERS} must be the servers to which you deployed the projects that you created in {PRODUCT}.
 
-.Prerequisites
-* {PRODUCT} {PRODUCT_VERSION} is installed. For installation options, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
-* One or more {KIE_SERVERS} are running and contain deployed project data. For instructions, see {URL_PACKAGING_DEPLOYING_PROJECT}[_{PACKAGING_DEPLOYING_PROJECT}_].
-* Entando App Builder 5.0.3 is installed. For instructions, see link:http://docs.entando.com/[_Getting Started with Entando App Builder_].
-* You are logged in to the Entando Admin Console.
+You can use the following links to configure Red Hat Process Automation Manager with App Builder:
```

Comment:
I'd reword to "For instructions on getting started with App Builder and for configuring {PRODUCT} with App Builder, see the following links:"

Reply:
Done